deps/linenoise/linenoise.c

```diff
@@ -708,6 +708,41 @@ void linenoiseEditMoveRight(struct linenoiseState *l) {
     }
 }
 
+/* People have different preferences on what is considered a word
+ * when editing commandline. For example, in command
+ * `set business:department:product value`,
+ * somne might treat the entire long key as a word, while others
+ * treat it as 3 subwords and prefer granular word jumping navigation
+ * to make fixing a typo (e.g. "business:deaprtment:product") easier.
+ * In order to accommodate this, we treat escape sequences
+ * ESC b/f and ESC[1;5D/C diffrently.
+ */
+static int isSubWordDelimiter(const char c) {
+    return !isalnum(c);
+}
+static int isBigWordDelimiter(const char c) {
+    return isspace(c);
+}
+
+static void linenoiseEditMoveWordLeft(struct linenoiseState *l, int (*isDelimiter)(char c)) {
```

Comment:
```suggestion
typedef void (*delimeterFn)(char c);

static void linenoiseEditMoveWordLeft(struct linenoiseState *l, isDelimiterFn isDelimiter ) {
```
nit: Minor preference for readability.

Reply:
Thanks! Done